webapp/web/spinner.css

```diff
@@ -9,7 +9,7 @@ Tobias Ahlin's Spinkit CSS (https://github.com/tobiasahlin/SpinKit/blob/master/s
   width: 17px;
   height: 17px;
   position: absolute;
-  bottom: -13px;
```

Comment:
Should be -13px otherwise the spinner is off the single indicator charts
![Screenshot 2020-08-31 at 12 47 02](https://user-images.githubusercontent.com/3750869/91707143-15ad9b00-eb88-11ea-9d33-dedc3254a158.png)

Reply:
Thanks for the catch! reverted to -13px